src/qibolab/instruments/zhinst.py

```diff
@@ -505,7 +505,9 @@ def register_couplerflux_line(self, coupler):
 
     def run_exp(self):
         """Compilation settings, compilation step, execution step and data retrival"""
+        # self.experiment.save("saved_exp")  # saving experiment
         self.exp = self.session.compile(self.experiment, compiler_settings=COMPILER_SETTINGS)
+        # self.exp.save("saved_exp")  # saving compiled experiment
         self.results = self.session.run(self.exp)
```

Comment:
Before merging, please drop dead code.

Reply:
i asked him to get those lines there to document how to save the experiments in the case Zurich supports asks for them. I would like to have them somewhere.

Reply:
Then I would suggest the docstring :)